apps/website/src/components/courses/GroupSwitchModal.tsx

```diff
@@ -352,31 +414,218 @@ const GroupSwitchModal: React.FC<GroupSwitchModalProps> = ({
                 {isSubmitting ? 'Submitting...' : 'Confirm'}
               </CTALinkOrButton>
             </div>
+          )}
 
-            {!isManualRequest && (
-            <div className="border-t border-gray-200 pt-4">
-              <div className="flex flex-col gap-2">
-                <h3 className="text-size-sm font-medium">Don't see a group that works?</h3>
-                <p className="text-size-sm text-[#666C80]">
-                  You can request a manual switch to join a group that's full or a group that is not
-                  listed above, and we'll do our best to accommodate you.
-                </p>
-                <CTALinkOrButton
-                  variant="secondary"
-                  onClick={() => setIsManualRequest(true)}
-                  aria-label="Request manual group switch"
-                >
-                  Request manual switch
-                </CTALinkOrButton>
-              </div>
+          {!isManualRequest && (
+          <div className="border-t border-color-divider pt-4">
+            <div className="flex flex-col gap-2">
+              <h3 className="text-size-sm font-medium">Don't see a group that works?</h3>
+              <p className="text-size-sm text-[#666C80]">
+                You can request a manual switch to join a group that's full or a group that is not
+                listed above, and we'll do our best to accommodate you.
+              </p>
+              <CTALinkOrButton
+                variant="secondary"
+                onClick={() => setIsManualRequest(true)}
+                aria-label="Request manual group switch"
+              >
+                Request manual switch
+              </CTALinkOrButton>
             </div>
-            )}
-          </form>
-        </>
+          </div>
+          )}
```

Comment:
One nice bit of cleanup:

You can move this up and into L396 above to combine all the UI for `!isManualRequest` and prevent the `isManualRequest` (L400
to L417) from interrupting the reader